kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -1309,6 +1322,19 @@ The various `CALL*` (and other inter-contract control flow) operations will be d
          <static> OLDSTATIC:Bool => OLDSTATIC orBool STATIC </static>
          <schedule> SCHED </schedule>
 
+    rule <k> #mkCall ACCTFROM ACCTTO ACCTCODE BYTES APPVALUE ARGS STATIC:Bool
+          => #touchAccounts ACCTFROM ACCTTO ~> #accessAccounts ACCTFROM ACCTTO ~> #loadProgram BYTES ~> #initVM ~> #precompiled?(ACCTCODE, SCHED) ~> #execute
+          ...
+          </k>
+          <useGas> false </useGas>
+          <callDepth> CD => CD +Int 1 </callDepth>
+          <callData> _ => ARGS </callData>
+          <callValue> _ => APPVALUE </callValue>
+          <id> _ => ACCTTO </id>
+          <caller> _ => ACCTFROM </caller>
+          <static> OLDSTATIC:Bool => OLDSTATIC orBool STATIC </static>
+          <schedule> SCHED </schedule>
```

Comment:
Why is this rule being duplicated in particular? It seems that the only change is that the gas cells aren't touched, but it should be fine to touch them I guess? They won't ever be compared against anyway, so it doesn't matter if we shuffle around the values in them.. If we don't duplicate this rule, do we still get the performance benefit? we want the minimal change that results in the performance benefit.

Reply:
It's duplicated because we do not want the values in `<gas>`, <callGas>`, and `<refund>` to change.